packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -263,6 +262,21 @@ export class BridgeHelper {
     )
   }
 
+  static getOutboundTransferData = async (
+    gatewayAddress: string,
+    provider: providers.Provider,
+    filter: ethers.providers.Filter = {}
+  ) => {
+    const contract = L1ERC20Gateway__factory.connect(gatewayAddress, provider)
+    const logs = await BridgeHelper.getEventLogs(
```

Comment:
This isn't type safe, we can instead use typechain interface for this

some sample code I used in a similar script
```
const event = bridge.l2Bridge.l2GatewayRouter.filters.OutboundTransferInitiated(null, user, null, null, null, null);

    const withdrawals = await bridge.l2Bridge.l2GatewayRouter.attach(l2Network.tokenBridge.l2ERC20Gateway).queryFilter(event, "0x0", "latest");
```

Reply:
yeah, let's do this update as a separate PR